src/libs/actions/Session/index.ts

```diff
@@ -85,21 +85,31 @@ Onyx.connect({
  */
 function signOut() {
     Log.info('Flushing logs before signing out', true, {}, true);
+    
+    // In case this is a supportal token, we won't have infinite sessions setup since the token will be
+    // short lived and we can just skip calling logout.
+    if (!isSupportalToken()) {
```

Comment:
this should be opposite and the method should be defined before signOut
```suggestion
    if (isSupportalToken()) {
```

Reply:
Nope, this is correct. If this is a supportal token, we can't do the logout call to our API

Reply:
sorry